homeassistant/components/overkiz/__init__.py

```diff
@@ -159,6 +175,45 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
 
+    async def handle_execute_command(service: ServiceCall) -> None:
```

Comment:
This looks it targeting entities so it should be an entity service and not an integration service?

Reply:
Thanks! I will look into this.